app/models/spree/payment_method/sisow_billing.rb

```diff
@@ -39,7 +44,9 @@ def process_response(response)
     end
 
     def start_transaction(transaction_type, opts = {})
-      @sisow_transaction = SisowTransaction.create(transaction_type: transaction_type, status: 'pending')
+      @sisow_transaction = SisowTransaction.create(
+        transaction_type: transaction_type,
+        status: 'pending')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.